From 884d8b9afe4d7eba262307570ecc29ff1b78be23 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Mon, 15 Aug 2022 07:20:19 -0400
Subject: [PATCH 2/3] Silicence warnigns about loading DLC in light mode

---
 deeplabcut/__init__.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/deeplabcut/__init__.py b/deeplabcut/__init__.py
index 3db1e1d..5219f1a 100644
--- a/deeplabcut/__init__.py
+++ b/deeplabcut/__init__.py
@@ -18,8 +18,6 @@ DEBUG = True and "DEBUG" in os.environ and os.environ["DEBUG"]
 from deeplabcut import DEBUG
 from deeplabcut.version import __version__, VERSION
 
-print(f"Loading DLC {VERSION}...")
-
 try:
     from deeplabcut import generate_training_dataset
     from deeplabcut import refine_training_dataset
@@ -32,9 +30,7 @@ try:
 
     from deeplabcut.utils.skeleton import SkeletonBuilder
 except (ModuleNotFoundError, ImportError):
-    print(
-        "DLC loaded in light mode; you cannot use any GUI (labeling, relabeling and standalone GUI)"
-    )
+    pass
 
 from deeplabcut.create_project import (
     create_new_project,
-- 
2.34.1

